src/Microsoft.Health.Dicom.SqlServer/Features/CustomTag/SqlCustomTagStore.cs

```diff
@@ -94,14 +94,15 @@ public async Task<IReadOnlyList<CustomTagStoreEntry>> GetCustomTagsAsync(string
                 {
                     while (await reader.ReadAsync(cancellationToken))
                     {
-                        (int tagKey, string tagPath, string tagVR, int tagLevel, int tagStatus) = reader.ReadRow(
+                        (int tagKey, string tagPath, string tagVR, string tagPrivateCreator, int tagLevel, int tagStatus) = reader.ReadRow(
                             VLatest.CustomTag.TagKey,
                             VLatest.CustomTag.TagPath,
                             VLatest.CustomTag.TagVR,
+                            VLatest.CustomTag.TagPrivateCreator,
```

Comment:
>VLatest.CustomTag.TagPrivateCreator [](start = 28, length = 35)

Won't this always be null?

Reply:
it is Null for standard tags, should be some value for private tags.
I'll update AddCustomTag api to require privateCreator for private tag.